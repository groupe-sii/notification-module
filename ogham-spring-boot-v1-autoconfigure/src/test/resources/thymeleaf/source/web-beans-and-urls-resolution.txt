<html xmlns:th="http://www.thymeleaf.org" th:inline="text" th:remove="tag">[[${@fakeService.hello(name)}]]

[[@{/fake/resources/foo.js}]]
[[${(#mvc.url('FC#hello').arg(0, 'world')).build()}]]
</html>