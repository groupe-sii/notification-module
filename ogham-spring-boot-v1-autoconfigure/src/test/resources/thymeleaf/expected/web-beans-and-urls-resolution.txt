hello world

/fake/resources/foo.js
/fake/hello/world
